app/__init__.py

```diff
@@ -678,13 +681,38 @@ def user_signup_fake():
   user.id
   return user.get_dict()
 
-@app.route("/images")
+@app.route("/imagesurls")
 def get_images():
     images = Image.query.all()
     output = []
     for image in images:
-      output.append(image.get_dict())
+      dictionary = image.get_dict()
+      dictionary['signedurl'] = create_presigned_url_local('reviews/' + dictionary['url'])
```

Comment:
i wonder if you can just put the signing of the URLs into the get_dict method itself